ecs-agent/tmds/handlers/taskprotection/v1/handlers/interface.go

```diff
@@ -0,0 +1,10 @@
+package handlers
+
+import (
+	"github.com/aws/amazon-ecs-agent/ecs-agent/api"
+	"github.com/aws/amazon-ecs-agent/ecs-agent/credentials"
+)
+
+type TaskProtectionClientFactoryInterface interface {
```

Comment:
Q: Since we moved the `TaskProtectionClinetFactoryInterface` and are already importing it from`ecs-agent` in this PR, could we remove it from https://github.com/aws/amazon-ecs-agent/blob/master/agent/handlers/agentapi/taskprotection/v1/handlers/interface.go?

Reply:
It is removed in this PR. The diff shows it as moved to `agent/vendor/github.com/aws/amazon-ecs-agent/ecs-agent/tmds/handlers/taskprotection/v1/handlers/interface.go`.
<img width="950" alt="Screenshot 2023-06-28 at 4 23 34 PM" src="https://github.com/aws/amazon-ecs-agent/assets/9619772/52cddff8-0133-4ed7-a18d-8fd7f0dff2bd">

Reply:
Ah I see, that was my miss. Thanks for clarifying!